policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/base.yaml

```
@@ -0,0 +1,9 @@
+description: Nova Scotia provides this base amount under the Age tax credit.
```

Let's just use `base` when there are multiple elements
```suggestion
description: Nova Scotia provides this amount under the Age Tax Credit.
```